environments/gcp/locals.tf

```diff
@@ -0,0 +1,13 @@
+locals {
+  application            = "flyte"
+  environment            = "terraform"
+  name_prefix            = "${local.application}-${local.environment}"
+  region                 = data.google_client_config.current.region
+  project_id             = "<my-GCP-project>" #Insert your GCP Project ID
+  workload_identity_pool = "${local.project_id}"
```

Comment:
I think this could be shielded from the users if we just activate it in the module configuring the gke cluster.